crates/pop-cli/src/commands/call/chain.rs

```diff
@@ -608,6 +613,65 @@ fn parse_function_name(name: &str) -> Result<String, String> {
 	Ok(name.to_ascii_lowercase())
 }
 
+/// Represents a node in the network with its RPC endpoints and chain properties.
+#[allow(dead_code)]
+#[derive(Debug)]
+pub(crate) struct RPCNode {
```

Comment:
Does this logic makes more sense to have it in a common folder? I can see this implemented for contracts at some point

Reply:
On a second thought, it makes total sense to move it. Done in [e3cd4f3](https://github.com/r0gue-io/pop-cli/pull/658/commits/e3cd4f384c8e219c05a4d40d49261b2de0df02ff).